pkg/generators/models/enums.go

```diff
@@ -0,0 +1,178 @@
+package models
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"text/template"
+
+	"github.com/getkin/kin-openapi/openapi3"
+
+	tpl "github.com/contiamo/go-base/pkg/generators/templates"
+)
+
+// GenerateEnums outputs the Go enum models with validators
+func GenerateEnums(specFile io.Reader, dst string, packageName string) error {
```

Comment:
I think we should define a struct type for the parameters which will have only the `packageName` for now. I'm pretty sure we would need to add some more and it would break the function signature.